distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -0,0 +1,33 @@
+priority "grant";
+
+// Double forward slashes "//" indicate comments and have no effect on policies.
+
+
+//                        Content Directory Monitor (CDM) Permissions
+// Adding required permissions for a new Content Directory Monitor.
+//
+// If the directory being monitored is located in DDFs home directory, then adding these permissions
+// is not required and these instructions can be ignored.
+//
+// Configuring CDM requires adding read and write permissions to the directory being monitored. For
+// each configured CDM, the following 2 permissions are required:
+//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read, write";
+// Replacing <DIRECTORY_PATH> with the path of the directory being monitored.
+//
+// Line 1 gives the CDM the permissions to read from the monitored directory path. Line 2 gives the
+// CDM the permissions to recursively read and write from the monitored directory path, specified
+// by the directory path's suffix "${/}-".
+//
+// Adding new permissions requires a system restart to take affect.
+//
+// Trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example,
+// adding a permission for "/test/path" and "/test/path/" are equivalent. The recursive forms
+// "/test/path${/}-", and "/test/path/${/}-" are also equivalent.
+//
+// If a CDM configuration is deleted, then the corresponding permissions that were added should be
+// deleted to avoid granting unnecessary permissions to parts of the system.
+//
+grant codeBase "file:/org.apache.camel.camel-core/catalog-core-directorymonitor/catalog-core-camelcomponent/com.google.guava/catalog-core-standardframework/org.apache.camel.camel-blueprint/catalog-core-urlresourcereader/catalog-transformer-resource/catalog-rest-endpoint/org.apache.cxf.cxf-rt-transports-http/javax.servlet-api/org.eclipse.jetty.servlet/platform-filter-delegate/platform-filter-clientinfo/platform-filter-response/security-filter-authorization/security-filter-login" {
```

Comment:
❓Do we need to put this here? Since we're not expecting anyone to change this, could we put this in the main policy file instead and only have the directory monitored in this file?

Reply:
✏️ Should we add a commented out example as a reference?

Reply:
Not sure what you mean by the first comment, but all the files listed require permissions for the directory being configured.